frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Logo from './Logo.png';
+const Navbar = () => {
+  // colors: {
+  //   c100: '#f9320c', // red
+  //   c200: '#141f32', // dark-blue
+  //   c300: '#7a8baa', // grayish-blue
+  //   c400: '#f9f7ff', // off-white
+  //   c600: '#8072fc', // light-blue
+  //   c700: '#ed3ce3', // purple
+  //   c800: '#74fa5e'  // light-green
+  // },
+  return (
```

Comment:
create static data array and loop through it instead of copying and pasting multiple times.